test/controllers/api/deploys_controller_test.rb

```diff
@@ -3,7 +3,11 @@
 SingleCov.covered!
 
 describe Api::DeploysController do
+  assert_route verb: "GET", path: "/api/deploys/active_count", to: 'api/deploys#active_count'
```

Comment:
@sandlerr thought it was useful. I agree too, it helps make sure you end up with the routes you intend.